source/scpd/tests/LayoutTest.d

```diff
@@ -0,0 +1,80 @@
+/*******************************************************************************
+
+    Contains runtime size & ABI object layout checks and
+    function argument passing in the D glue layer works properly.
+
+    Copyright:
+        Copyright (c) 2019 BOS Platform Foundation Korea
+        All rights reserved.
+
+    License:
+        MIT License. See LICENSE for details.
+
+*******************************************************************************/
+
+module scpd.tests.LayoutTest;
+
+import scpd.tests.GlueTypes;
+import scpd.types.Stellar_SCP;
+import scpd.types.Stellar_types;
+
+import std.digest.sha;
+import std.random;
+import std.stdio;
+import std.string;
+import std.traits;
+
+/// Contains the size and offset of a field within a struct
+extern(C++) struct FieldInfo
+{
+    long size;
+    long offset;
```

Comment:
So... Is `long` mangled correctly ? 🤔

Reply:
Yeah long seems fine. But not `unsigned lang lang`.